ole.log(entity);

```diff
@@ -0,0 +1,62 @@
+warning: LF will be replaced by CRLF in exchanges/graphcache/src/operations/invalidate.ts.
```

Comment:
Accidental commit?

Reply:
Hmm, `eslint` and windows seemed to disagree there did not notice that my bad